### PR TITLE
add latency to terminal window

### PR DESCRIPTION
**Pull request by SuperDuperAlon**
## Summary by CodeRabbit

- **Style**
	- Enhanced positioning, styling, and responsiveness of various elements like `.windows-container`, `.search-container`, and `.gate` in the CSS file.
- **Bug Fixes**
	- Rectified positioning logic in the `Gate` class for improved alignment.
- **Documentation**
	- Updated display logic for statistics to only exhibit round trip time information.

### CODE REVIEW

**Review by yaelih — COMMENTED** [review]
Looks great!

File: css/terminal7.css [hunk 1]
```diff
@@ -897,4 +888,17 @@ video {
     opacity: 0.8;
     z-index: 3;
 }
-
+@media (max-width: 600px) {
```
Comment (daonb) [inline css/terminal7.css:891]: Why did you move this section here?

File: src/gate.ts [hunk 2]
```diff
@@ -448,7 +448,7 @@ export class Gate {
             container.style.removeProperty("height")
             scale = 1
             container.style.left = "0"
-            container.style.top = "22px"
+            container.style.top = "0"
```
Comment (daonb) [inline src/gate.ts:451]: Is this line necessary? We already have top: 0 in the css

File: css/terminal7.css [hunk 3]
```diff
@@ -767,11 +749,20 @@ audio { display: none; }
     text-align: left;
     padding-left: 1.5em;
     text-shadow: none;
+    z-index: 20;
```
Comment (daonb) [inline css/terminal7.css:752]: It seems that gate-stats is also used in the map and it changes it.  You should move your changes to `.gate > .gate-stats` bellow
image

File: css/terminal7.css [hunk 4]
```diff
@@ -767,11 +749,20 @@ audio { display: none; }
...
+    border-bottom: 1px rgba(255, 255, 255, 0.7) solid;
```
Comment (daonb) [inline css/terminal7.css:753]: Let's use the current border color of we use the separate buttons on the bnav bar: rgba(55, 55, 2, 1)

File: css/terminal7.css [hunk 5]
```diff
@@ -767,11 +749,20 @@ audio { display: none; }
...
+    border-right: 0;
+    border-top: 0;
+    -webkit-border-top-right-radius: 6px;
```
Comment (daonb) [inline css/terminal7.css:756]: This and the next line are not needed anymore. It's an ancient hack and all modern browsers have support for `border-top-right-radius`. In the future, use https://caniuse.com to check if you need any browser specific rules

**Review by daonb — CHANGES_REQUESTED** [review]
Thanks! It's almost there...

File: css/terminal7.css (hunk 4 above)
Comment (SuperDuperAlon) [reply css/terminal7.css:753]: Will change it
the root color is --dark-border

File: css/terminal7.css (hunk 5 above)
Comment (SuperDuperAlon) [reply css/terminal7.css:756]: It's not clear, what lines we're talking about
- 4th line will be changed to border-top-right-radius
- lines 2-3 can be removed

**Comment by daonb** [comment]
Thanks! I merged manually

**Pull request closed without merging**